uqcsbot/scripts/dominos.py

```diff
@@ -7,40 +7,43 @@
 from uqcsbot.utils.command_utils import loading_status, UsageSyntaxException
 import requests
 
-MAX_COUPONS = 10 # Prevents abuse
+MAX_COUPONS = 10  # Prevents abuse
```

Comment:
Move this comment above to follow standard

Reply:
Comments on the same line are used frequently. Should I change all of them?

Reply:
Not sure what Brae was referring to. Inline comments are fine as long as there are two spaces before the hash.